wigwam/_docker_distrib.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import os
+from textwrap import dedent
+
+
+def distrib_dockerfile(
+    base: str,
+    source_tag: str,
+    source_path: os.PathLike[str] | str,
+    distrib_path: os.PathLike[str] | str,
+    libdir: str,
+) -> str:
+    """
+    Returns a Dockerfile for a distributable build.
+
+    The distributable image includes the installed software and all of its runtime
+    dependencies, but excludes build-time dependencies. The working directory on the
+    generated image will also be moved to the location given at `distrib_path`.
+
+    Parameters
+    ----------
+    base : str
+        The base image tag.
+    source_tag : str
+        The tag of the image on which the project is installed.
+    source_path : os.PathLike[str] or str
+        The installation path of the project on the source image.
+    distrib_path : os.PathLike[str] or str
+        The desired installation path on the distributable image.
+    libdir : str
+        The base name of the directory where object libraries are stored (e.g. "lib" or
+        "lib64"). This should match the value of `LIBDIR` in CMake's `GNUInstallDirs`
+        module.
+        See https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.
+
+    Returns
+    -------
+    dockerfile: str
+        The generated Dockerfile.
+    """
+    dockerfile: str = dedent(
+        f"""
+            FROM {source_tag} as source
+
+            FROM {base}
+
+            # Change user to root - this is necessary to enable file copying.
+            #USER root
+
```

Comment:
Rather than commenting these lines out, let's simply remove them.

```suggestion
```

Reply:
I had already done this and it was sitting on an un-pushed commit on my machine 😆